extra-dev/packages/coq-tactician/coq-tactician.8.11.dev/opam

```diff
@@ -1,43 +1,51 @@
 opam-version: "2.0"
-name: "coq-tactician"
-synopsis: "Tactician: A Seamless, Interactive Tactic Learner and Prover for Coq"
+version: "8.11.dev"
+synopsis:
+  "Tactician: A Seamless, Interactive Tactic Learner and Prover for Coq"
 description: """
-  Tactician is a tactic learner and prover for the Coq Proof Assistant.
-  The system will help users make tactical proof decisions while they retain
-  control over the general proof strategy. To this end, Tactician will learn
-  from previously written tactic scripts, and either gives the user suggestions
-  about the next tactic to be executed or altogether takes over the burden of
-  proof synthesis. Tactician's goal is to provide the user with a seamless,
-  interactive, and intuitive experience together with strong, adaptive proof
-  automation.
-"""
+Tactician is a tactic learner and prover for the Coq Proof Assistant.
+The system will help users make tactical proof decisions while they retain
+control over the general proof strategy. To this end, Tactician will learn
+from previously written tactic scripts, and either gives the user suggestions
+about the next tactic to be executed or altogether takes over the burden of
+proof synthesis. Tactician's goal is to provide the user with a seamless,
+interactive, and intuitive experience together with strong, adaptive proof
+automation."""
+maintainer: ["Lasse Blaauwbroek <lasse@blaauwbroek.eu>"]
+authors: ["Lasse Blaauwbroek <lasse@blaauwbroek.eu>"]
 homepage: "https://coq-tactician.github.io"
-dev-repo: "git+https://github.com/coq-tactician/coq-tactician"
 bug-reports: "https://github.com/coq-tactician/coq-tactician/issues"
-maintainer: "Lasse Blaauwbroek <lasse@blaauwbroek.eu>"
-authors: "Lasse Blaauwbroek <lasse@blaauwbroek.eu"
 depends: [
   "ocaml" {>= "4.08"}
-  "dune" {>= "2.5"}
+  "dune" {>= "2.5" & build}
```

Comment:
```suggestion
  "dune" {>= "2.5"}
```
Please do not use `build` for Dune. It won't work. See issues in the Dune repo for a long discussion of why.